guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/test/java/org/guvnor/asset/management/client/editors/project/structure/ReleaseRepositoryPresenterTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2014 JBoss Inc
```

Comment:
In NetBeans IDE, I have the following template for Java Class files:
```
<#assign licenseFirst = "/*">
<#assign licensePrefix = " * ">
<#assign licenseLast = " */">
<#include "${project.licensePath}">
<#if package?? && package != "">
package ${package};

</#if>
public class ${name} {

}
```
The first four lines should also appear in templates for other file types like unit tests, enumerations, exceptions etc. Templates can be edited in Tools > Templates menu.

There are pre-configured license headers that can be viewed in project properties. If the project effective POM contains license element that matches one of the headers, it is used. Organization (the copyright owner) is also picked up from POM and year is generated correctly, too.